src/i965_drv_video.h

```diff
@@ -69,7 +69,9 @@
 #define DEFAULT_SATURATION      50
 
 #define ENCODER_QUALITY_RANGE     2
+#define ENCODER_QUALITY_RANGE_AVC     8
```

Comment:
Yes, these kernels do support 1-7 and we need to support them for customer use cases.